packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -1250,3 +1253,102 @@ test('bid zero', async t => {
     },
   );
 });
+
+test('auctioneer chainStorage RPC API snapshot', async t => {
+  //#region copied from on-exit test
```

Comment:
I'm not a fan of #region comments.

Reply:
agreed that this one, at least, doesn't carry its weight. As noted above, I aim to "DRY out copy-pasta from on-exit test".

Reply:
I didn't find a straightforward way to DRY out the common stuff; I changed the `#region` comments into a function.

Reply:
it occurs to me that, rather than repeating a bunch of stuff from another test, I can just add the `t.snapshot()` into that test.